crates/rolldown/src/stages/link_stage/tree_shaking.rs

```diff
@@ -118,17 +117,13 @@ fn include_symbol(ctx: &mut Context, symbol_ref: SymbolRef) {
     canonical_ref = namespace_alias.namespace_ref;
   }
 
-  // TODO(hyf0): suspicious why need `USED_AS_NAMESPACE`
   let is_namespace_ref = canonical_ref_owner.namespace_object_ref == canonical_ref;
   if is_namespace_ref {
-    ctx.used_exports_info_vec[canonical_ref_owner.id].used_info |= UsedInfo::USED_AS_NAMESPACE;
+    ctx.used_exports_info_vec[canonical_ref_owner.id].used_info |= UsedInfo::USED_AS_NAMESPACE_REF;
   }
 
-  // TODO(hyf0): why we need `used_symbol_refs` to make if the symbol is used?
```

Comment:
https://github.com/rolldown/rolldown/blob/d96752bac77e949b34384a54bb3fcf616bc64d86/crates/rolldown/src/stages/generate_stage/compute_cross_chunk_links.rs#L274-L276, we include all chunk meta imports here, but some symbol ref is not included anymore, if we don't filter them,  code would panic here https://github.com/rolldown/rolldown/blob/d96752bac77e949b34384a54bb3fcf616bc64d86/crates/rolldown/src/stages/generate_stage/compute_cross_chunk_links.rs#L277-L286, because unused symbol ref will not have `chunk_id`